### PR TITLE
LB-2347: Update terminology for listings API

### PR DESCRIPTION
**Pull request by ndougherty-va**
Changed business entity to business location.

Changed paths:
`businessEntityListings` to `listingSyncListings`
`businessEntityListingScores` to `listingScore`

@vendasta/insync 
@richard-rance 
@jredl-va

### CODE REVIEW

**Review by vendastaluan — APPROVED** [review]

**Review by dhanesh-kapadiya — APPROVED** [review]

File: openapi/listings/listings.yaml [hunk 1]
```diff
@@ -13,23 +13,23 @@ servers:
...
         - OAuth2:
             - listing
       x-lifecycle:
-        status: proposed
+        status: trustedTester
       description: |-
-        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
-        Gets a list of listings for a business entity
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        Gets a list of listings for a business location. This corresponds to the Listing Sync page in Listing Builder.
```
Comment (richard-rance) [inline openapi/listings/listings.yaml:32]: Can we clarify the type of listings?

```suggestion
        Gets a list of web directory listings for a business location. This corresponds to the Listing Sync page in Listing Builder.
```

File: openapi/listings/listings.yaml [hunk 2]
```diff
@@ -148,16 +148,14 @@ components:
...
       description: |-
-        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
-        Gets a list of listings for a business entity
-      x-tags:
-        - Business Entity Listings
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        A list of listings for a business location
       x-internal: false
       properties:
         id:
```
Comment (richard-rance) [inline openapi/listings/listings.yaml:161]: In the schema `listingSyncListings` you will need to update the `default` and `enum` options for the `type` field. It should match the path. (`default: listings` becomes `default: listingSyncListings`)

Comment (richard-rance) [reply openapi/listings/listings.yaml:161]: Also they are missing from the `listingScores` schema.

File: openapi/listings/listings.yaml [hunk 3]
```diff
@@ -82,25 +82,25 @@ paths:
...
           description: Business Not Found
         '406':
           description: No Listing Product Active
       tags:
         - Options
-        - Business Entity Listings
+        - Listing Sync Listings
     parameters: []
-  '/businessEntityListingScores/{id}':
+  '/listingScore/{id}':
```
Comment (richard-rance) [inline openapi/listings/listings.yaml:101]: This should be plural.
```suggestion
  '/listingScores/{id}':
```

**Review by richard-rance — CHANGES_REQUESTED** [review]
There are a few changes to make in the OAS. You also need to update the guides to match the new paths.

File: openapi/listings/listings.yaml (hunk 3 above)
Comment (richard-rance) [reply openapi/listings/listings.yaml:101]: Should the score be prefixed with Listing Sync to match the listings? 

```suggestion
  '/listingSyncScores/{id}':
```

Comment (ndougherty-va) [reply openapi/listings/listings.yaml:101]: Not according to the meeting minutes https://vendasta.jira.com/browse/LB-2347

File: openapi/listings/listings.yaml (hunk 1 above)
Comment (ndougherty-va) [reply openapi/listings/listings.yaml:32]: We dont only deal in web directories, we also do GPS, aggregators, etc

Comment (richard-rance) [reply openapi/listings/listings.yaml:32]: Is there some collective term for them? "Digital directory"? If not that is fine.

File: openapi/listings/listings.yaml [hunk 4]
```diff
@@ -401,6 +401,9 @@ components:
           description: Vendasta's unique ID for the business
         type:
           type: string
+          default: listingScore
```
Comment (richard-rance) [inline openapi/listings/listings.yaml:404]: Again this should be plural.

Comment (ndougherty-va) [reply openapi/listings/listings.yaml:404]: It's a get endpoint tho, and there is only 1 score per business

Comment (ndougherty-va) [reply openapi/listings/listings.yaml:404]: On the other hand we do offer a breakdown of the score.

**Review by richard-rance — APPROVED** [review]

**Pull request merged**